### ISSUE TITLE
False positive for \ReflectionException

### ISSUE THREAD

**Issue by marcospassos** [pepakriz/phpstan-exception-rules#33]
First of all, thank you for bringing such a fantastic library!

I found a bug today catching a reflection exception:

```php
try {
     new \ReflectionProperty(Foo::class, 'property');
} catch (\ReflectionException $exception) {
   // invalid report
}
```

The inspection message is:
```
 ReflectionException is never thrown in the corresponding try block  
```

It's wrong since `ReflectionProperty` throw an exception if the property or class does not exist.

**Comment by pepakriz** [comment]
Hi, fixed in master

Thank you!
